apps/dashboard/src/app/knowledge-box/knowledge-box-users/users-manage/users-manage.service.ts

```diff
@@ -14,7 +14,18 @@ export class UsersManageService {
         switchMap(([kb, account]) => kb.getUsers(account.slug)),
       ),
     ),
-    shareReplay(),
+    shareReplay(1),
+  );
+
+  invitesKb = this._onUpdateUsers.pipe(
+    switchMap(() => this.sdk.currentKb),
```

Comment:
don't you need a `take(1)` on `currentKB` to avoir problems when switching KB?

Reply:
Fixed